subworkflows/local/utils_nfcore_rnafusion_pipeline/main.nf

```diff
@@ -156,16 +156,7 @@ def validateInputParameters() {
     genomeExistsError()
 
     if (params.no_cosmic) {
-            log.warn("Skipping COSMIC DB download from `FUSIONREPORT_DOWNLOAD` and skip using it in `FUSIONREPORT`")
-    }
-
-    if (params.starfusion_build && !params.fusion_annot_lib) {
```

Comment:
Why removing this warning?

Reply:
Because that parameter wasn't used anymore

Reply:
Would you like me to reimplement it?

Reply:
No, i just checked that fusion_annot_lib has "exists": true, but could we have a check for fusion_cannot_lib in build_references when building STARFUSION?

Reply:
Just to make sure it is defined?

Reply:
Good idea!

Reply:
I added the extra check :)